### DISCUSSION TITLE
How to send file using php webhook

### DISCUSSION THREAD

**Discussion by Diego-Schmidt** [vasani-arpit/WBOT#229] [Q&A]
Hello and thanks in advance for the help. Is there a way to send a file using php webhook? I didn't saw any axample on how to do it if it is possible at all.

**Comment by Diego-Schmidt** [comment]
Do you send files in php webhook with something like this ? $response = array(array('file' => 'base64 code', 'type' => 'file'));

**Reply by vasani-arpit** [reply]
You want to send file from webhook to whatsapp. right? or you want to send from whatsapp to webhook?

**Reply by Diego-Schmidt** [reply]
From webhook to whatsapp, maily images

**Reply by vasani-arpit** [reply]
WBOT does send the base64 of the image in `base64DataFile` property of the payload. you can do `var_dump` on the entire payload to find out.

**Reply by elfata92** [reply]
Please give an example?
